Santillana-Demo-AR-Experiencia-Residuos/ARSceneViewConstans.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  ARSceneViewConstans.swift
+//  Santillana-Demo-AR-Experiencia-Residuos
+//
+//  Created by Carlos Daniel Hernandez Chauteco on 6/13/19.
+//  Copyright © 2019 Alejandro Mendoza. All rights reserved.
+//
+
+import Foundation
+
+class ARSceneViewConstans {
+    static let share = ARSceneViewConstans()
+    
+    let labelPlaneDetectionFirstMessage = "Escanea tu cuarto para iniciar"
```

Comment:
Can you change the constants names? I think something like 'startScanningMessage' would be more suitable